src/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface PdfMetadata {
+   title: string;
+   subject: string;
+   keywords: string;
+   author: string;
+   creator: number;
+   producer: number;
+   creationdate: number;
+   moddate: number;
+   tagged: string;
+   form: string;
+   pages: number;
+   encrypted: string;
+   page_size: string;
+   mediabox: string;
+   cropbox: string;
+   bleedbox: string;
+   trimbox: string;
+   artbox: string;
+   file_size: string;
+   optimized: string;
+   pdf_version: number;
+}
+
+export interface PdfLib {
```

Comment:
This is the generic type to support your injection. It's pretty much saying "I accept anything that has a least a `info` and `pdfToText` function" which is what you want instead of `any`